aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Learn how to use the ASP.NET Core SignalR HubContext service for sending notifications to clients from outside a hub.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+# Send messages from outside a hub
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+
+The SignalR hub is the core abstraction for sending messages to clients connected to the SignalR server. It's also possible to send messages from other places in your app using the `IHubContext<T>` service. This article explains how to access a SignalR `IHubContext<T>` to send notification to clients from outside a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Get an instance of `IHubContext<T>`
+
+In ASP.NET Core SignalR, you can access an instance of `IHubContext<T>` via dependency injection. You can inject an instance of `IHubContext<your_hub_type>` into a controller, middleware, or other DI service. Use the instance to send messages to clients.
```

Comment:
`your_hub_type`? What kind of Java is this? :) Just keep it as `MyHub` or something that looks .NETy

Reply:
See the other comment now. Let's discuss this in that thread.

Reply:
@anurse It's intended to look out-of-place, so that the user knows to replace it. Past experience has shown that using a .NETy custom type leads to questions like "Where is MyHub defined?". If we decide that `MyHub` is better, we'll need verbiage stating it should be replaced.